### DISCUSSION TITLE
Question about permission

### DISCUSSION THREAD

**Discussion by Trung15010802** [natsuk4ze/gal#193] [Q&A] [answered]
In android i don't need to use:
```
// Check Permission
await Gal.hasAccess();

// Request Permission
await Gal.requestAccess();
```

and my app works fine. I'm confused about it. Am I need this to save video or image and my image/video can be found on Gallery when publish my app on google play store?

**Comment by natsuk4ze** [comment, marked as answer]
Hi @Trung15010802  👋

### Whether or not an permission request is required depends on the version of Android.

If your app only supports Android API 30 or higher, you don't need to request permissions, but if not, you do. 
Please see [wiki](https://github.com/natsuk4ze/gal/wiki/Permissions#android) for more details.

### Finally, please don't forget to "Mark as answer ✅". Thank you.
